source/module_base/mymath3.cpp

```diff
@@ -32,24 +34,24 @@ void heapAjust(double *r, int *ind, int s, int m)
 
 void heapsort(const int n, double *r, int *ind)
```

Comment:
Does the program using it? This implementation seems to be not reliable, and I strongly suggest removing this method, replacing with `std::sort`.